pkg/pillar/cmd/tpmmgr/tpmmgr_test.go

```diff
@@ -222,3 +227,43 @@ func TestSealUnseal(t *testing.T) {
 		t.Errorf("Seal/Unseal operation failed, want %v, but got %v", dataToSeal, unsealedData)
 	}
 }
+
+func TestSealUnsealMismatchReport(t *testing.T) {
+	_, err := os.Stat(etpm.TpmDevicePath)
+	if err != nil {
+		t.Skip("TPM is not available, skipping the test.")
+	}
+
+	rw, err := tpm2.OpenTPM(etpm.TpmDevicePath)
+	if err != nil {
+		t.Errorf("OpenTPM failed with err: %v", err)
+		return
```

Comment:
isn't this the same as:
```suggestion
		t.Fatalf("OpenTPM failed with err: %v", err)
```

Reply:
it is , but all the are tests are written like that, I thought it might be intentional, so I just followed the pattern.

Reply:
okay, fine